src/App.tsx

```diff
@@ -1,24 +1,47 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+
+import { getTodos } from './api/todos';
+import { Content } from './components/Content';
+import { Errors } from './components/Errors';
 import { UserWarning } from './UserWarning';
 
-const USER_ID = 0;
+import { Error } from './types/Error';
+import { Todo } from './types/Todo';
+
+const USER_ID = 6192;
```

Comment:
Don't hardcode the userId
The user should log in first 
<img width="1268" alt="image" src="https://user-images.githubusercontent.com/99403198/220877774-88c71353-c26f-46c8-b052-49166f85b599.png">